### PR TITLE
{lang}[system/system] Java v16.0.1, Java v16

### PR DESCRIPTION
**Pull request by Flamefire**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Flamefire** [comment]
Test report by @Flamefire
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
taurusa6 - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2603 v4 @ 1.70GHz (broadwell), Python 2.7.5
See https://gist.github.com/076343ea411bbefd14d2951da2d55ac1 for a full test report.

File: easybuild/easyconfigs/j/Java/Java-16.eb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Contribution from the Crick HPC team / UL HPC Team
+# uploaded by J. Sassmannshausen
+# updated to version 202
```
Comment (Micket) [inline easybuild/easyconfigs/j/Java/Java-16.eb:3]: I think we can drop this comment

**Review by Micket — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/j/Java/Java-16.eb (hunk 1 above)
Comment (Flamefire) [reply easybuild/easyconfigs/j/Java/Java-16.eb]: Thanks

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
alvis-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/ad841ea3bfb35c0c5f2ff6618dfb6317 for a full test report.

**Comment by Micket** [comment]
Going in, thanks @Flamefire!

**Pull request merged**